content/v2.0/visualize-data/variables/create-variable.md

```diff
@@ -16,7 +16,8 @@ There are multiple variable types that provide different means of populating you
 _For information about variable types, see [Variable types](/v2.0/visualize-data/variables/variable-types/)._
 
 {{% note %}}
-##### Reserved keywords
+##### Restrictions on variable names
```

Comment:
- Variable name restrictions
- Make sure that any links pointing to this heading (`/v2.0/visualize-data/variables/create-variable/#reserved-keywords`) are updated (there may not be any, we just need to make sure).